web-wa-sqlite/README.md

```diff
@@ -127,6 +121,73 @@ const ExampleComponent = () => {
 }
 ```
 
+## Migrating the app
+
+ElectricSQL supports migrating the back-end while the app is running.
+Migrations are additive-only.
+For example, while running the app we may connect to the Postgres backend and add a column `other_value` to the `items` table:
+```shell
+docker exec -it -e PGPASSWORD=password local-stack-postgres_1-1  psql -h 127.0.0.1 -U postgres -d electric
+electric=# ALTER TABLE items ADD other_value TEXT;
+WARNING:  assigning automatic migration version id: 20230529120539_974
+ALTER TABLE
+```
+
+This migration will automatically be picked up by Electric and will be streamed to the application
+which will migrate its local SQLite database.
+
+Then, remains to migrate the code of our application to do something with the new column.
```

Comment:
I think we can be more precise here. 
Is migrating the code the best way of saying it? Then, I think we could say that what we actually are doing is to migrate the type model to know about the new column.